### ISSUE TITLE
Train loss drops, but val loss goes up with denoising_diffusion_pytorch_1d

### ISSUE THREAD

**Issue by cherylLbt** [lucidrains/denoising-diffusion-pytorch#149]
Hey, 

I use the `pred_x0` mode and the default parameters. I trained the network using a time series dataset. The training loss keeps going down smoothly, while the val loss was very small at the first two epochs while it soars suddenly at the 3rd epoch. Then the val loss whipsawed instead of dropping when the training loss goes down. I really look forward to help with hyper-parameter settings and problem-solving. Thanks for any potential help in advance.

**Comment by Adrian744** [comment]
This looks like a overfitting problem. Try to increase/decrease your learning rate and batch sizes. There is no "perfect" answer, it is always trial and error.

**Comment by kirilzilla** [comment]
I do not see any explicit calculation or logging of a validation loss in the code?
